srcpkgs/omegat/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'omegat'
+pkgname=omegat
+version=4.3.3
+revision=1
+depends="openjdk-jre"
```

Comment:
this should be either `virtual?java-runtime` or a specific openjdk#-jre version, not this metapackage